wallet/wallet.c

```diff
@@ -658,7 +658,7 @@ static bool wallet_ever_used(struct wallet *w)
 	if (db_get_intvar(w->db, "bip32_max_index", 0) != 0)
 		return true;
 
-	/* Or if they do a unilateral close, the output to us provides a UTXO. */
+	/* Or if they do a unilateral close, the output to us provides an UTXO. */
```

Comment:
That sounds wrong: the 'y' sound at the beginning of UTXO makes this an 'a' not an 'an', and google seems to agree....